data/processed/defra/mean_pm10.yaml

```diff
@@ -0,0 +1,18 @@
+API: DEFRA
+API documentation: https://uk-air.defra.gov.uk/data/pcm-data
+Dataset code: mean_pm10_air_pollution
+endpoint: https://uk-air.defra.gov.uk/data/pcm-data#pm10
+description: Mean PM10 particulate background pollution data at NUTS 2 regions aggregated from 1km x 1km resolution data modelled by DEFRA.
+nuts_version: 2016
+schema:
+  nuts_id:
+    type: string
+    description: NUTS2 (2016) region code
+  mean_pm10:
+     type: float
+     description: mean pm10 particulate pollution value
+     unit: Gravimetric Units

```

Comment:
maybe here we could do:

```yaml
unit:
  name: gravimetry
  ref: https://en.wikipedia.org/wiki/Gravimetry
  type: acceleration
  value: m/s^2
```

Later on we can have a type `gravimetry_SI` described in the unified schema that would be:

```yaml
...
units:
  - ...
  - gravimetry_SI:
    ref: https://en.wikipedia.org/wiki/Gravimetry
    type: acceleration
    value: m/s^2
  - ...
...
```

so that if we find multiple types of `gravimetry` or `acceleration` we unified and maybe convert to [SI](https://github.com/nestauk/imt_playbook/wiki/Acronyms)